core/src/main/java/com/graphhopper/util/exceptions/ConnectionNotFoundException.java

```diff
@@ -25,7 +30,75 @@
  * @author Peter Karich
  */
 public class ConnectionNotFoundException extends DetailedIllegalArgumentException {
+    private static final String INTRO = "Connection between ";
+    private static final String FMT = "point %d and point %d";
+    private static final String JOIN = ", and ";
+    private static final String OUTRO = " not found";
+
     public ConnectionNotFoundException(String var1, Map<String, Object> details) {
         super(var1, details);
     }
+
+    public ConnectionNotFoundException(List<Path> paths, PointList waypoints) {
+        super(unconnectedListString(paths), unconnectedInfo(paths, waypoints));
```

Comment:
Wouldn't it be better to call `this(unconnectedListString(paths), unconnectedInfo(paths, waypoints));` in this case?

Reply:
Yes absolutely, that was an oversight.